crates/wdl-engine/src/cache/hash.rs

```diff
@@ -330,6 +335,13 @@ impl Hashable for Struct {
     }
 }
 
+impl Hashable for EnumVariant {
+    fn hash(&self, hasher: &mut Hasher) {
+        ValueKind::EnumVariant.hash(hasher);
+        self.value().hash(hasher);
```

Comment:
This is, perhaps, controversial. I'm interested in what you think, @peterhuene.

Reply:
So I think we need to also hash the variant's name in addition to it's value.

The name is important in case they coerce the variant value to a string (or interpolate).

The value is important in case the variant's constant expression changes between runs.